### DISCUSSION TITLE
Thank You / seperate activation off universal forwards and backwards

### DISCUSSION THREAD

**Discussion by MrGreenPullover** [linearmouse/linearmouse#248] [Ideas]
Hallo,

first of all, thank You very much for this piece of very helpful software, it makes using the Mac so much more special. 
Out of the box, the mouse point speed and accelaration is perfect, I did not even dare to bother with the settings.
You deserved Yourself a coffee ;)  (by MGP)

The only issue I have is, well, me and my strange behavior ;)
On my PC I use the the side buttons for backwards as intended but for forwards I set it to middle click.
Because I'm not normal... :D

So, my hope is, there could be a way to get separate checkboxes to activate universal forwards and backwards separately, so I could use Your backwards on the back button, but the misinterpretation of the macOS that interpreted the side buttons by default as middle click, as I wanted...

**Comment by lujjjh** [comment]
Thank you very much for your support!

Advanced options to enable universal back and forward separately have been shipped in v0.7.4-beta.2.

To enable back only, you may

1. Enable universal back and forward in the LinearMouse Preference.

1. Open the config file with your favorite text editor, such as TextEdit.app or VS Code.

<img width="607" alt="image" src="https://user-images.githubusercontent.com/3000535/191879711-46035aa3-425a-4c46-863e-8e74e3780cc7.png">

2. Replace all

```
"universalBackForward" : true
```

with

```
"universalBackForward" : "backOnly"
```

and save it.

3. Reload the config.

<img width="452" alt="image" src="https://user-images.githubusercontent.com/3000535/191879991-b6330efe-5909-41a8-93a5-2df2fa90f9ef.png">

**Comment by MrGreenPullover** [comment]
Thank You very much, now it's working perfect for me, indeed!

Am Do., 22. Sept. 2022 um 18:23 Uhr schrieb Jiahao Lu <
***@***.***>:
